Makefile

```diff
@@ -68,12 +68,11 @@ generate-setup: ## Current workarround to generate the testdata with the correct
 test: ## Run the go tests ($ go test -v ./cmd/... ./pkg/...)
 	go test -v ./cmd/... ./pkg/...
 
-.PHONY: test-project-generation
-test-project-generation: ## Run the unit tests (used in the CI)
-	- go get sigs.k8s.io/kind@v0.5.1
-	./test.sh
+.PHONY: test-ci
+test-ci: ## Run the unit tests (used in the CI)
```

Comment:
> test-ci

Just call it `test`. You can run it locally, so it may be confusing to call it test-ci.

Reply:
I agree. The name test-ci is because of the prow. I will change but, then, it means that the prow will be broken. @alexeldeib could you help us by changing there to merge this one?